internal/cluster/common.go

```diff
@@ -66,6 +66,48 @@ func updateClusterStatus(log logrus.FieldLogger, db *gorm.DB, clusterId strfmt.U
 	return cluster, nil
 }
 
+func updateClusterProgressDB(db *gorm.DB, clusterId strfmt.UUID, extra ...interface{}) (*common.Cluster, error) {
```

Comment:
What's the purpose of this method?  Looks like a generic method for updating a cluster record - nothing to do with progress as its name suggests.  Why not use UpdateCluster()?

Reply:
Done.